docs/tutorials/update_value.rst

```diff
@@ -0,0 +1,25 @@
+##############################
+Update value in PI
```

Comment:
Thanks for adding an explanatory tutorial! Looks good to me!

Reply:
thanks to @ldariva